tests/reporter/kubernetes.go

```diff
@@ -320,7 +320,7 @@ func (r *KubernetesReporter) logDMESG(virtCli kubecli.KubevirtClient, logsdir st
 			commands := []string{
 				virt_chroot.GetChrootBinaryPath(),
 				"--mount",
-				virt_chroot.GetChrootMountNamespace(),
+				virt_chroot.GetChrootNSMountPath(),
```

Comment:
The last problem I see is that this won't work for you environment.  Up to you if you want to a address it

Reply:
This is fine. We don't do development or testing under this environment. thanks.